tools/vulture_suppressions/cdp.py

```diff
@@ -0,0 +1,4 @@
+from cdp import CDP
+
+# not really used any more, but unit tests still use CDP
```

Comment:
...for now ;)

Reply:
Yes, that's something I want to discuss with you tomorrow -- I can do a quick thing, or perhaps #20706 is within reach next week.